package.json

```diff
@@ -134,6 +134,7 @@
     },
     "dependencies": {
         "azure-arm-resource": "7.3.0",
+        "azure-arm-website": "5.7.0",
```

Comment:
Do we really need this dependency? this will bring a lot of additional dependencies (although I do not see any change in package-lock.json) and we should be very careful with new dependencies.
I think we are using this package to update config/metadata/deployments. can we not do this via direct rest calls (like we do in PE service or tasks)

Reply:
We can see in package.lock.json that this package doesn't bring a lot of dependencies. IT requires the same packages which were being used in azure-arm-rest.

This was also helping in keeping the code cleaner and required less involvement in things like selecting api-version for different api's.